src/helpers.js

```diff
@@ -1,18 +1,19 @@
-import { isError, attempt, reduce, isObject, isUndefined } from 'lodash'
+import { reduce, isObject, isUndefined } from 'lodash'
 import dotenv from 'dotenv'
 import fs from 'fs'
 import chalk from 'chalk'
 
-export const readAppjson = filePath => {
-  // eslint-disable-next-line global-require
-  const appjson = attempt(() => require(filePath))
-  return isError(appjson) ? undefined : appjson
-}
+const attemptPath = (filePath, callback) => (
+  fs.existsSync(filePath) ? callback(filePath) : undefined
+)
 
-export const readDotEnv = filePath => {
-  const config = attempt(() => dotenv.parse(fs.readFileSync(filePath)))
-  return isError(config) ? undefined : config
-}
+export const readAppjson = filePath => (
+  attemptPath(filePath, () => require(filePath)) // eslint-disable-line global-require
```

Comment:
Is attemptPath maybe an unneeded abstraction? Makes it more tricky to understand what gets returned when from readAppjson if it doesn't exists.

`fs.existsSync(filePath) ? require(filePath) : undefined`
`fs.existsSync(filePath) ? dotenv.parse(fs.readFileSync(filePath)) : undefined`